unic/ucd/normal/tests/conformance_tests.rs

```diff
@@ -67,7 +67,7 @@ fn test_decomposition_type_conformance() {
                     .unwrap_or(low);
                 low..(high + 1)
             };
-            let exp_dt = Some(get_dt_from_name(fields[1]));
+            let exp_dt = Some(fields[1].parse().expect("Invalid Decomposition Type"));
```

Comment:
There are probably other places where the new APIs can clean up tests; I didn't go over everything with a fine tooth comb. I only really got the ones in this PR because they popped up as a compilation error.